src/sinks/sematext/logs.rs

```diff
@@ -103,6 +103,10 @@ impl SinkConfig for SematextLogsConfig {
     fn sink_type(&self) -> &'static str {
         "sematext_logs"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
This sink appears to just wrap the `elasticsearch` sink which, I think, means it should support acknowledgements since the `elasticsearch` sink does?

Reply:
Fixed in 392697054